app/src/androidTest/java/ch/epfl/polychef/FullRecipeImageTest.java

```diff
@@ -0,0 +1,111 @@
+package ch.epfl.polychef;
```

Comment:
This class is a duplicate of the refactored class in _images_, this one was changed thus you need to add your changes to the one in the _images_ package